client/app/components/Input/InputTextarea.scss

```diff
@@ -34,4 +34,28 @@
   p {
     margin: $size-0;
   }
-}
\ No newline at end of file
+}
+
+.dark {
+  .button {
+    color: $white;
+    b, strong {
+      color: $white !important;
+    }
+    &Selected {
+      color: $key-lime;
+      b, strong {
+        color: $key-lime !important;
+      }
+    }
+  }
+  .content {
+    color: $white;
+    a {
+      color: $cornflower !important;
+    }
+    b, strong {
+      color: $white !important;
```

Comment:
Any way we can avoid using `!important` here and a few other spots in this file? Would prioritize heavier specificity over using these, but no worries if it's necessary!

Reply:
Good call out and I agree with you. Unfortunately, I have to override this with `!important` because of Pell Editor womp womp.

Reply:
Pell Editorrrrrrrr womp.